demisto_sdk/commands/validate/validate_manager_v2_README.md

```diff
@@ -0,0 +1,117 @@
+* **-a, --validate-all**
+Whether to run all validation on all files or not.
+* **-i, --input**
+The path of the content pack/file to validate specifically.
+* **-pc, --post-commit**
+Whether the validation should run only on the current branch's committed changed files. This applies only when the **-g** flag is supplied.
+* **-st, --staged**
+Whether the validation should run only on the current branch's staged files. This applies only when the **-g** flag is supplied.
+* **--prev-ver**
+Previous branch or SHA1 commit to run checks against.
+* **--no-multiprocessing**
+Run validate all without multiprocessing, for debugging purposes.
+* **-j, --json-file**
+The JSON file path to which to output the command results.
+* **--category-to-run**
+Run specific validations by stating category they're listed under in the config file.
+* **-af, --allow-fix**
+Wether to autofix failing validations with an available auto fix or not.
+* **--config-path**
+Path for a config file to run, if not given - will run the default path at: ...
+
+**Examples**:
+
+`demisto-sdk validate --prev-ver SHA1-HASH`
+This will validate only changed files from the branch given (SHA1).
+<br><br>
+
+`demisto-sdk validate --post-commit`
+This indicates that the command runs post commit.
+<br><br>
+
+`demisto-sdk validate -i Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml`
+This will validate the file Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml only.
+<br><br>
+
+`demisto-sdk validate -a`
+This will validate all files under `Packs` directory
+<br><br>
+
+`demisto-sdk validate -i Packs/HelloWorld`
+This will validate all files under the content pack `HelloWorld`
+<br><br>
+
+### Notes
+* In external repositories (repos which contain the `.private-repo-settings` file in its root) **all** the validations are ignorable.
+
+
+### Error Codes and Ignoring Them
+Each error found by validate  has an error code attached to it - the code can be found in brackets preceding the error itself.
+For example: `path/to/file: [IN103] - The type field of the proxy parameter should be 8`
+
+The first 2 letters indicate the error type and can be used to easily identify the cause of the error.
+| Code | Type |
+| --- | --- |
+| BA | Basic error |
+| BC | Backwards compatibility error |
+| CJ | Conf json error |
+| CL | Classifier error |
+| DA | Dashboard error |
+| DB | DBootScore error |
+| DO | Docker error |
+| DS | Description error |
+| ID | Id set error |
+| IF | Incident field or type error |
+| IM | Image error |
+| IN | Integration or script error |
+| IT | Incident type error |
+| MA | Mapper error |
+| PA | Pack files error (pack-metadata, pack-secrets, pack-ignore) |
+| PB | Playbook error |
+| RM | Readme error |
+| RN | Release notes error |
+| RP | Reputation error |
+| SC | Script error |
+| ST | Structure error |
+| WD | Widget error |
+
+
+Each user will have a personal config files which he can edit however he wants.
```

Comment:
Lets provide a link to our default file to view, and say "If this default configuration isn't desirable, then you can override by doing x" we should push for the default behavior